src/pages/CardPage.js

```diff
@@ -408,6 +406,7 @@ const CardPage = ({ card, history, versions, draftedWith, cubedWith, synergistic
             {filteredVersions.length > 0 ? (
               <PagedList
                 pageSize={10}
+                // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
The `pageWrap` value could be extracted into a separate component to fix the warning.